homeassistant/components/devolo_home_control/binary_sensor.py

```diff
@@ -97,3 +110,48 @@ def _sync(self, message=None):
         else:
             _LOGGER.debug("No valid message received: %s", message)
         self.schedule_update_ha_state()
+
+
+class DevoloRemoteControl(DevoloDeviceEntity, BinarySensorEntity):
+    """Representation of a remote control within devolo Home Control."""
+
+    def __init__(self, homecontrol, device_instance, element_uid, key):
+        """Initialize a devolo remote control."""
+        self._remote_control_property = device_instance.remote_control_property.get(
```

Comment:
Same comment as https://github.com/home-assistant/core/pull/38386 (for a new PR)